### PR TITLE
User database updated

### PR DESCRIPTION
**Pull request by PrasantiNalla**
added 3 new columns userBio, userType and profileimageurl
image

### CODE REVIEW

File: WhaleSpotting/Models/Database/WhaleSighting.cs [hunk 1]
```diff
@@ -10,10 +10,10 @@ public class WhaleSighting
...
-    public string PhotoImageURL { get; set; }
+    public string? PhotoImageURL { get; set; }
     public int NumberOfWhales { get; set; }
     public ApprovalStatus ApprovalStatus {get;set;}
-    public string Description { get; set; }
-    public WhaleSpecies WhaleSpecies { get; set; }
-    public User User { get; set; }
+    public string? Description { get; set; }
+    public WhaleSpecies? WhaleSpecies { get; set; }
+    public User? User { get; set; }
```
Comment (francescastocco) [inline WhaleSpotting/Models/Database/WhaleSighting.cs:18]: Do we want these values to be nullable in the database?

File: WhaleSpotting/Models/Database/WhaleSpecies.cs
Comment (francescastocco) [inline WhaleSpotting/Models/Database/WhaleSpecies.cs:1]: Same with these, why should they be nullable?

File: WhaleSpotting/WhaleSpottingDbContext.cs [hunk 2]
```diff
@@ -6,9 +6,9 @@ namespace WhaleSpotting;
 
 public class WhaleSpottingDbContext : DbContext
 {
-    public DbSet<User> Users { get; set; }
-    public DbSet<WhaleSighting> WhaleSightings { get; set; }
-    public DbSet<WhaleSpecies> WhaleSpecies { get; set; }
+    public DbSet<User>? Users { get; set; }
+    public DbSet<WhaleSighting>? WhaleSightings { get; set; }
+    public DbSet<WhaleSpecies>? WhaleSpecies { get; set; }
```
Comment (francescastocco) [inline WhaleSpotting/WhaleSpottingDbContext.cs:11]: Did you mean to add `?` here?

**Review by francescastocco — CHANGES_REQUESTED** [review]
The User model looks good, but unsure why the the fields have been made nullable

File: WhaleSpotting/WhaleSpottingDbContext.cs (hunk 2 above)
Comment (mharkins99) [reply WhaleSpotting/WhaleSpottingDbContext.cs]: Nullable status changed

File: WhaleSpotting/Models/Database/WhaleSpecies.cs
Comment (mharkins99) [reply WhaleSpotting/Models/Database/WhaleSpecies.cs:1]: Nullable status changed

File: WhaleSpotting/Models/Database/WhaleSighting.cs (hunk 1 above)
Comment (mharkins99) [reply WhaleSpotting/Models/Database/WhaleSighting.cs]: Nullable status changed

**Comment by mharkins99** [comment]
Multiple nullable values changed to non-nullable

File: WhaleSpotting/Migrations/20230323150939_updateDB.cs
Comment (francescastocco) [inline WhaleSpotting/Migrations/20230323150939_updateDB.cs:1]: This migration is empty so we should be able to delete this file

File: WhaleSpotting/Migrations/20230323150939_updateDB.Designer.cs
Comment (francescastocco) [inline WhaleSpotting/Migrations/20230323150939_updateDB.Designer.cs:1]: We should be able to delete this file

**Pull request closed without merging**